phing/build.yml

```diff
@@ -42,7 +42,7 @@ cm:
   strategy: config-split
   core:
     # The parent directory for configuration directories, relative to the docroot.
-    path: ../config
+    path: ${repo.root}/config
```

Comment:
This was done to ensure that a command run from outside of Drupal VM would work within Drupal VM. Are you sure this doesn't work? @arknoll recently made a related change: https://github.com/acquia/blt/pull/1305/files

Also, this does seem to work in our CI process: https://github.com/acquia/blt/blob/8.x/scripts/blt/ci/internal/run_tests.sh#L30